### PR TITLE
Create chiamaka-udechukwu.md

### PR DESCRIPTION
**Pull request by Chiamakabrowneyes**

### CODE REVIEW

**Review by PraiseOgwuche — COMMENTED** [review]
Quite succinct change made. Would have liked to see more diverse usage like writing in italics, titles etc.

LGTM.

**Comment by Chiamakabrowneyes** [comment]
> Quite succinct change made. Would have liked to see more diverse usage like writing in italics, titles etc.
> 
> LGTM.

Aha you make a great point!! I'd make those edits

File: chiamaka-udechukwu.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hey hey hey. Its Chiamakaa
```
Comment (Shadimrad) [inline chiamaka-udechukwu.md:1]: There's much more to know about amazing Chiamakaa; I attest!

**Review by PraiseOgwuche — APPROVED** [review]
Could show more dexterity but LGTM
Chiamakaaa!!
Your jollof rice and me; 5&6

**Review by Shadimrad — COMMENTED** [review]
How about using ">" to make the meanings in the quote format? (just a suggestion.)

**Pull request closed without merging**